config/controller/admin/enhancer.config.php

```diff
@@ -36,16 +36,23 @@
                 'required',
             ),
         ),
-        'format' => array(
-            'label' => __('Format:'),
-            'form' => array(
-                'type' => count($formats) === 1 ? 'hidden' : 'select',
-                'options' => $formats,
-                'value' => $current,
-            ),
-        ),
     ),
     'preview' => array(
         'view' => 'noviusos_slideshow::admin/enhancer/preview',
     ),
 );
+
+// Don't display list of format if we only have one format
+if (count($formats) > 1) {
+    $config['fields']['format'] = array(
+        'label' => __('Format:'),
+        'form'  => array(
+            'type'    => count($formats) === 1 ? 'hidden' : 'select',
```

Comment:
Why keeping this? You explicitely add this when "count > 1".
The other question is : if it was hidden, why removing it when count == 1 ? 
(except for avoiding the display of the label, which should be handled indeed).